matching-notification-api/src/main/java/org/phenotips/matchingnotification/storage/internal/DefaultMatchStorageManager.java

```diff
@@ -180,6 +180,7 @@ public boolean saveLocalMatches(List<PatientMatch> matches, String patientId)
                         match.setFoundTimestamp(existingMatch.getFoundTimestamp());
                         match.setComment(existingMatch.getComment());
                         match.setNotificationHistory(existingMatch.getNotificationHistory());
+                        match.setNotes(existingMatch.getNotes());
```

Comment:
At this point I wonder if we should remove all setXXX() methods (which are only ever used here, and may be misleading to programmer-users since everywhere else updateXXX() methods should be used) and have one clone() or transfer() method which takes care of this particular use case